src/views/partials/section/about.hbs

```diff
@@ -5,18 +5,10 @@
                 <h2 class="section-title"> DevConf</h2>
                 <br>
                 <p>
-                    Acreditamos que o conhecimento é livre, e essa crença nos inspirou a organizar o DevConf!
-                    O DevConf é um evento de tecnologia sem fins lucrativos organizado pelo Opensanca, a maior comunidade ativa
-                    de tecnologia do interior de São Paulo, em parceria com a UFSCar e IFSP São Carlos.
-                    A proposta desta edição do evento é colocar em discussão temas atuais de tecnologia como Docker, Ionic, Agile,
-                    Microserviços, JavaScript, Clean Architecture e Programação Funcional.
-                    Iremos promover apresentações com alto nível de conhecimento e relevância e, estimular a troca de
-                    experiência e network entre a comunidade de tecnologia do interior, que é caipira mas ama e se move por tecnologia!
-                    O evento irá acontecer no dia 05 de Maio de 2017, no Anfiteatro Bento Prado da Universidade
-                    Federal de São Carlos (UFSCar) em São Carlos/SP. Esperamos um público aproximado de 200 pessoas,
-                    entre estudantes, profissionais de tecnologia e entusiastas da região!
+                    {{info.about}}
```

Comment:
Nice! So lembrando que se a intenção for rolar uns HTML aqui é preciso por `{{{ info.about }}}`

Reply:
Boa.. vou alterar.. melhor ja deixar preparado